interfaces/builtin/docker_support.go

```diff
@@ -62,6 +62,11 @@ const dockerSupportConnectedPlugAppArmorUserNS = `
 userns,
 `
 
+const dockerSupportConnectedPlugAppArmorMqueue = `
+# allow unrestricted use of posix message queues
+mqueue,
```

Comment:
Do we need to check if the apparmor feature exits? I wonder as I've seen https://github.com/snapcore/snapd/blob/master/interfaces/builtin/posix_mq.go#L119C1-L122C1 . cc @alexmurray

Reply:
We do, this snippet is used conditionally.